src/routes/album.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express')
+const {createAlbum, getAlbums, getAlbumById, updateAlbumById, patchAlbumById, deleteAlbumById} = require('../controllers/album')
+const albumRouter = express.Router()
+
+
+albumRouter.post('/artists/:id/albums', createAlbum)
```

Comment:
This is the tricky route to define as the base route is `/artists` but the controller is an album one. There are a few options of how to handle this. My preference is to add this to the artist router file instead and the import the create album controller into that file. (If you do make the changes I have suggested regarding all the other routes then you will not be able to leave this route here as it will break) If you need help with any of this let me know